catalogs/catalog-jdbc-oceanbase/src/main/java/org/apache/gravitino/catalog/oceanbase/operation/OceanBaseTableOperations.java

```diff
@@ -58,10 +94,134 @@ protected String generateCreateTableSql(
       Distribution distribution,
       Index[] indexes) {
     if (ArrayUtils.isNotEmpty(partitioning)) {
-      throw new UnsupportedOperationException("Currently not support Partition tables.");
+      throw new UnsupportedOperationException(
+          "Currently we do not support Partitioning in oceanbase");
+    }
+
+    if (!Distributions.NONE.equals(distribution)) {
+      throw new UnsupportedOperationException("OceanBase does not support distribution");
+    }
+
+    validateIncrementCol(columns, indexes);
```

Comment:
Exactly the same as MySQL, it is recommended to be placed in the super class.

Reply:
Refactored.